.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!--Please ensure the PR fulfills the following requirements! -->
+### Pull Request Checklist:
+- [ ] This PR addresses an already opened issue (for bug fixes / features)
+    - This PR fixes issue #xyz
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Documentation has been added / updated (for bug fixes / features)
+- [ ] HISTORY.rst has been updated (with summary of main changes)
+- [ ] `bumpversion minor` has been called on this branch
```

Comment:
`bumpversion`, update `HISTORY.rst` is for a release ... but not necessary for each pull-request?

Reply:
Ah, I should have specific `bumpversion patch`. I know for us, it makes the release process much easier when the history and documentation is all up-to-date at time of release. By bumping the patch, the version will automatically be in `beta` and users/developers who are working off of the dev branch can also know easily what features are now available. In order to perform a `release`, `bumpversion release` will drop the `-beta`.

This approach is different but it also is part of the reason why our documentation has expanded greatly, as everyone who contributes a PR is expected to work on the documentation for their features.

Reply:
Hi @Zeitsperre and @cehbrecht, I will accept this release but would like to discuss the PR/release procedure in more detail when we meet on Thursday. We have been developing a release procedure but there are aspects that we would do differently from the `xclim` approach - but maybe we can compromise so that we have a common approach.

Sorry for the breaking change in `roocs-utils` - we didn't spot that one and we are fixing it.

Reply:
Not a problem. Feel free to merge and revise as needed. We'll talk Thursday!